raiden_contracts/tests/fixtures/contracts.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+import logging
+from web3.utils.events import get_event_data
+from eth_utils import is_address
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def contracts_manager():
+    """Reimplement this - fixture should return raiden-lib's contract manager.
```

Comment:
We already have the `contracts_manager` fixture in https://github.com/raiden-network/raiden-contracts/blob/a17d6d908fa85d9cf963f81e8fc0e7a76aaa33fb/raiden_contracts/tests/fixtures/contract_manager.py#L6, this one should be removed.
`raiden-lib's contract manager` - the contract manager should only stay in this repo.